modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/leases/LeaseTracker.java

```diff
@@ -88,38 +95,16 @@ public LeaseTracker(MetaStorageManager msManager) {
         this.msManager = msManager;
     }
 
-    /** Recoveries state from Vault and subscribers on further updates. */
-    public void startTrack() {
-        inBusyLock(busyLock, () -> {
+    /**
+     * Recovers state from Vault and subscribes to future updates.
+     *
+     * @param recoveryRevision Revision from {@link MetaStorageManager#recoveryFinishedFuture()}.
+     */
+    public CompletableFuture<Void> startTrackAsync(long recoveryRevision) {
```

Comment:
I believe that it's the same question, so please fill free to answer only in first topic. But why it's async?

Reply:
> But why it's async?

Because it returns the future, if you insist, I’ll remove the `Async` postfix.